Peer.Connectors/AzdoPullRequestSource.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Peer.Domain;
+using Peer.Domain.Models;
+using Peer.User;
+
+namespace Peer.Connectors
+{
+    class AzdoPullRequestSource : IPullRequestSource
+    {
+        private readonly ConfigModel _config;
```

Comment:
nit: ConfigModel isn't really a great name. Is this Azure config? Config for everything? Should it be AppConfig? Should it be AzureConfig? looking at it from the name alone I don't really know